src/ingest_validation_tools/table-schemas/level-2/wgs.yaml

```diff
@@ -12,14 +12,13 @@ fields:
 - name: gdna_fragmentation_quality_assurance
   description: Is the gDNA integrity good enough for WGS? This is usually checked
     through running a gel. TODO - Is this a boolean?
-- name: dna_assay_input
+- name: dna_assay_input_value
   description: Amount of DNA input into library preparation
   type: number
 - name: dna_assay_input_unit
   description: Units of DNA input into library preparation
   constraints:
-    enum:
-      - nuclei
+  enum: ug
```

Comment:
@cebriggs7135 : In this file, indenting matters -- It doesn't know that `enum` belongs to `constraints` unless it's indented. I'll fix -- no worries.

Reply:
... and since we want a list of options, even if there's only one, need to keep the dash.